### ISSUE TITLE
Issue with 'swift_versions' in plugin version 1.0.1

### ISSUE THREAD

**Issue by andriy-gulak-ew** [GigaDroid/flutter_udid#18]
During the production build for ios, I've faced with the following issue:

```
[!] Failed to load 'flutter_udid' podspec: 
    [!] Invalid `flutter_udid.podspec` file: undefined method `swift_versions=' for #<Pod::Specification name="flutter_udid">
    Did you mean?  swift_version=
                   swift_version.
     #  from /Users/vagrant/git/ios/.symlinks/plugins/flutter_udid/ios/flutter_udid.podspec:20
     #  -------------------------------------------
     #    s.ios.deployment_target = '8.0'
     >    s.swift_versions = ['4.0', '4.2', '5.0']
     #  end
     #  -------------------------------------------
```

The problem is clearly with this line:
`     >    s.swift_versions = ['4.0', '4.2', '5.0'] as described above.`

I've checked the 1.0.0 version - it works without any problems.

**Comment by oncul** [comment]
Same problem! I solved with comment swift version line

**Comment by GigaDroid** [comment]
I just published version 1.1.0 of the plugin, which should fix these errors.
